src/Controllers/LoginController.php

```diff
@@ -43,7 +52,7 @@ public function handle(ServerRequestInterface $request): ResponseInterface
          */
         $user = User::findOrFail($id);
 
-        $this->assertCan($request->getAttribute('actor'), 'flagrowCanImpersonate', $user);
+        $this->assertCan($request->getAttribute('actor'), 'fofCanImpersonate', $user);
```

Comment:
In relation to the above comment, I think this might not work as intended. Either the permission must be prefixed in the policy, or be unprefixed here.

Reply:
Odd, it appears to work when I tested locally, but will update anyway :)